run-kiosk-browser-forever-and-log.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# go to directory where this file is located
+cd "$(dirname "$0")"
+
+if [ "${VX_MACHINE_TYPE}" = bmd ]; then
+    URL=http://localhost:3000/speech-loader.html
+else
+    URL=http://localhost:3000/
+fi
+
+while true; do
+    echo "starting kiosk-browser"
+    URL="${URL}" bash ./run-kiosk-browser.sh
```

Comment:
Using the environment variable here is a little odd. Just make `run-kiosk-browser.sh` a proper script that accepts a URL via `$1`?